src/main/kotlin/no/nav/familie/ba/mottak/task/VurderLivshendelseTask.kt

```diff
@@ -116,9 +113,57 @@ class VurderLivshendelseTask(
         return personerMedSak
     }
 
-    private fun opprettVurderLivshendelseOppgave(fagsakPerson: RestFagsakDeltager,
-                                                 task: Task,
-                                                 beskrivelse: String): Boolean {
+    private fun opprettEllerOppdaterDødsfallshendelseOppgave(
+        fagsakPerson: RestFagsakDeltager,
+        task: Task
+    ): Boolean {
+        val (nyopprettet, oppgave) = hentEllerOpprettNyOppgaveForPerson(fagsakPerson, BESKRIVELSE_DØDSFALL)
+        val pdlPersonData = pdlClient.hentPerson(fagsakPerson.ident, "hentperson-relasjon-dødsfall")
+        val dødsfallBarnList = pdlPersonData.forelderBarnRelasjon.filter {
+            it.relatertPersonsRolle == FORELDERBARNRELASJONROLLE.BARN &&
+                pdlClient.hentPerson(
+                it.relatertPersonsIdent,
+                "hentperson-relasjon-dødsfall"
+            ).dødsfall.firstOrNull()?.dødsdato != null
+        }
+        val tekstBruker = if (pdlPersonData.dødsfall.firstOrNull()?.dødsdato != null) "bruker" else ""
+        val tekstOg = if (tekstBruker.isNotEmpty() && dødsfallBarnList.isNotEmpty()) " og " else ""
+        val tekstBarn = if (dødsfallBarnList.count() > 1) dødsfallBarnList.count()
+            .toString() + " barn" else if (dødsfallBarnList.isNotEmpty()) "barn" else ""
+        val tekstBarnIdenter = dødsfallBarnList.fold("") { identList, it -> "$identList ${it.relatertPersonsIdent}" }
+        val beskrivelsesTekst = "$BESKRIVELSE_DØDSFALL: " + tekstBruker + tekstOg + tekstBarn + tekstBarnIdenter
```

Comment:
Maybe we can move this to a function called lagDødsfallOppgaveBeskrivelse, similar to to lagUtflyttingOppgaveBeskrivelse?